colorama/ansitowin32.py

```diff
@@ -21,6 +21,23 @@ def is_a_tty(stream):
     return hasattr(stream, 'isatty') and stream.isatty()
 
 
+def check_windows_ansi_support():
+    try:
+        import winreg
```

Comment:
There is a problem here because winreg only exists in Python 3 but we support Python 2 too. Also, this function fails on older versions of Windows because the ReleaseID value doesn't exist in the registry. I am reluctant to pull this function - too much platform specific code has a high chance of breaking things.